### PR TITLE
Mahdiyeh/Feature: add close banner to EU and UK clients

### PR DESCRIPTION
**Pull request by mahdiyeh-fs**

### CODE REVIEW

File: src/banners/banners.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div id='close_banner_container' class='container invisible'>
+    <div class='close_banner_text'>
+        <img class='close_banner_img' src="images/banner/icon_left.png"/>
+        <div class='close_banner_text_wrapper'>
+            <h3>Binary.com is moving to Deriv 0n 30 November</h3>
```
Comment (akmal-binary) [inline src/banners/banners.html:5]: It looks like there is a digit `0` instead of `O` in Deriv 0n.

Comment (akmal-binary) [reply src/banners/banners.html:5]:
```suggestion
            <h3>Binary.com is moving to Deriv on 30 November</h3>
```

Comment (akmal-binary) [reply src/banners/banners.html:5]: Also, should it have `th` after the date? Like 30th November?

File: src/banners/banners.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            margin: 0 40px 0 auto;
+            padding: 5px 16px;
+            line-height: 32px;
+            color: #fff;
+            border-radius: 4px;
+            background-color:#008000;
+            font-weight: normal;
+            text-align: center;
+        }
+        &_btn a {
```
Comment (akmal-binary) [inline src/banners/banners.scss:58]: Nitpick: We can move this inside `&_btn` on top

File: src/index.html [hunk 3]
```diff
@@ -78,6 +79,18 @@
             </div>
         </div>
     </div>
+    <div id='close_banner_container' class='container invisible'>
+        <div class='close_banner_text'>
+            <img class='close_banner_img' src="v<version>/images/banner/icon_left.png"/>
+            <div class='close_banner_text_wrapper'>
+                <h3>Binary.com is moving to Deriv 0n 30 November</h3>
```
Comment (akmal-binary) [inline src/index.html:86]: Same here

File: src/index.js [hunk 4]
```diff
@@ -225,6 +225,7 @@ function processFooter(selected_language_name) {
                 var isEu = isEuCountry(data.landing_company);
                 var FOOTER_TEXT = isEu ? FOOTER_TEXT_EU : FOOTER_TEXT_NON_EU;
     
+                showBanner(isEu)
```
Comment (akmal-binary) [inline src/index.js:228]:
```suggestion
                showBanner(isEu);
```

Comment (mahdiyeh-fs) [reply src/index.js]: sure, thanks, fixed.

**Review by akmal-binary — APPROVED** [review]

File: src/common/util.js [hunk 5]
```diff
@@ -515,3 +515,12 @@ function getBinaryUrl(page) {
 
     return binary_url;
 }
+
+function getderivUrl(page) {
```
Comment (matin-binary) [inline src/common/util.js:519]: You can use a CamelCase for function name.

**Review by matin-binary — APPROVED** [review]

**Review by matin-binary — APPROVED** [review]

**Review by alamin-deriv — APPROVED** [review]

**Pull request merged**